test/unit/lib/static/components/controls/run-button.jsx

```diff
@@ -116,7 +116,7 @@ describe('<RunButton />', () => {
 
         it('should switch to "Run checked tests" if there are checked tests', () => {
             selectorsStub.getCheckedTests.returns([{testName: 'testName', browserName: 'browserName'}]);
-            const component = mkConnectedComponent(<RunButton />, {
+            mkConnectedComponent(<RunButton />, {
```

Comment:
does this line do anything?

Reply:
Yes. It calls a stub that is used in assert inside the component.